file-formats/cota/type/zif_aff_cota_v1.intf.abap

```diff
@@ -28,13 +41,14 @@ INTERFACE zif_aff_cota_v1
       communication_type          TYPE ty_comm_type,
       "! <p class="shorttext">Communication Target Class</p>
       "! Communication target class
-      communication_target_class     TYPE zif_aff_types_v1=>ty_object_name_30,
+      communication_target_class  TYPE zif_aff_types_v1=>ty_object_name_30,
       "! <p class="shorttext">Allow Multiple Application Destinations</p>
       "! Allow multiple application destinations
       allow_multiple_destinations TYPE abap_bool,
```

Comment:
Boolean are rendered as a checkbox and we recommend to put checkboxes at the bottom of the UI. Meaning shuffle `allow_multiple_destinations` and `multitenancey_mode` here, unless the actual order in the UI is important to you.